app/components/search-result.js

```diff
@@ -27,6 +27,30 @@ export default Ember.Component.extend(Analytics, {
         return result.description.slice();
     }),
 
+    shortContributorList: Ember.computed('result', function() {
+        let nameArr = [];
+        let numNames = 0;
+        let filtered = this.get('result.contributors').filter(this.isBibliographic);
+        for (let i = 0; i < filtered.length; i++) {
+            nameArr[i] = filtered[i];
+            numNames++;
+            if (numNames === 10 || i === filtered.length - 1) {
+                if (numNames < filtered.length)
+                    nameArr[i].originalLength = filtered.length;
```

Comment:
It seems like you're using the presence of this (numeric) property as a boolean flag to indicate that this item is the last item in the list, *and* that there are more results?

Is there a more explicit way to do this, for clarity? Perhaps a separate (boolean) computed that compared the lengths of the two arrays?